TabMonConfigBuilder/Helpers/Host.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Collections.Generic;
+using System.Text;
+
+namespace TabMonConfigBuilder.Helpers
+{
+    /// <summary>
+    /// Object defining a host.
+    /// </summary>
+    class Host
+    {
+        private IDictionary<string, Process> Processes { get; set; }
+        private string HostName { get; set; }
+        private string HostString = "      <Host computerName=\"{0}\" address=\"{0}\" specifyPorts=\"true\">\r\n{1}      </Host>";
+
+        public Host(string host, string processName, string process, int processNum, int portNum)
```

Comment:
could this be:
public Host(string host, Process process)

Reply:
Talked to Patrick offline. For clarity we decided to keep it as is.